src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -190,48 +192,97 @@ public CaseDetails updateCase(String authorization,
      * @return the associated {@link CaseData} if the case is submitted
      */
     public CaseDetails submitCase(String authorization, CaseRequest caseRequest)
-        throws PdfServiceException, CaseDocumentException {
+        throws PdfServiceException {
         CaseData caseData = assignCaseToLocalOfficeService.convertCaseRequestToCaseDataWithTribunalOffice(caseRequest);
-        CaseDetails caseDetails = triggerEventForSubmitCase(authorization, caseRequest);
+        CaseDetails caseDetails = getCaseDetailsWithCaseRefNumberFromEcm(caseRequest, authorization);
+
+        //  Create case pdf file(s). If the user selected language is Welsh, then we also create Welsh pdf file
+        //  and add it to our pdf files list
         caseData.setEthosCaseReference(caseDetails.getData().get("ethosCaseReference") == null ? "" :
                                            caseDetails.getData().get("ethosCaseReference").toString());
-        caseData.setReceiptDate(caseDetails.getData().get("receiptDate") == null ? "" :
-                                    caseDetails.getData().get("receiptDate").toString());
-        caseData.setFeeGroupReference(caseDetails.getData().get("feeGroupReference") == null ? "" :
-                                          caseDetails.getData().get("feeGroupReference").toString());
-        List<PdfDecodedMultipartFile> acasCertificates = null;
-        try {
-            acasCertificates = pdfService.convertAcasCertificatesToPdfDecodedMultipartFiles(
-                caseData, acasService.getAcasCertificatesByCaseData(caseData));
-        } catch (AcasException e) {
-            log.error("Failed to connect to ACAS service", e);
-        } catch (InvalidAcasNumbersException e) {
-            log.error("Invalid ACAS numbers", e);
-        }
 
         UserInfo userInfo = idamClient.getUserInfo(authorization);
         List<PdfDecodedMultipartFile> casePdfFiles =
             pdfService.convertCaseDataToPdfDecodedMultipartFile(caseData, userInfo);
-        List<DocumentTypeItem> documentList = caseDocumentService
-            .uploadAllDocuments(authorization, caseRequest.getCaseTypeId(), casePdfFiles, acasCertificates);
 
-        if (caseData.getClaimantRequests().getClaimDescriptionDocument() != null) {
-            documentList.add(caseDocumentService.createDocumentTypeItem(
-                OTHER_TYPE_OF_DOCUMENT,
-                caseData.getClaimantRequests().getClaimDescriptionDocument()
-            ));
-        }
+        //  Submit e-mail to the user with generated ET1 pdf file attached. The pdf file is the user selected
+        //  contact language (Welsh or English)
+        PdfDecodedMultipartFile pdfFile = ENGLISH_LANGUAGE.equals(CaseServiceUtil.findClaimantLanguage(caseData))
```

Comment:
if the language is null/empty, welsh template will be used. May be use other way around, check for welsh language and if not, use English template as default